gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -287,7 +287,7 @@ def configure_interfaces(self, context, resource_data):
         """
 
         mgmt_ip = resource_data['mgmt_ip']
-
+        import time;time.sleep(15)
```

Comment:
Instead of putting such a big sleep, can we retry here with smaller sleep intervals ? we could save some secs in performance.. alternate good way is to try this API in a separate thread, if there is any other API that can also continue in parallel ?? This sort of sleep will effect latency and launch time.